.github/workflows/release.yml

```
@@ -0,0 +1,43 @@
+name: Triggered Release
+on: [workflow_dispatch]
+
+env:
+  OSS_USERNAME: ${{ secrets.OSS_USERNAME }}
+  OSS_PASSWORD: ${{ secrets.OSS_PASSWORD }}
+  GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Should this be `refs/heads/master`?

```suggestion
    if: github.ref == 'refs/heads/master'
```